lib/render.js

```diff
@@ -134,7 +134,13 @@ function getImage(source, image, options, x, y, callback) {
                     }
                 }
                 return callback(null, buffer);
-            });
+            }
+
+            if (options.format == 'utf') {
+                view.encode(options, encodeCallback);
+            } else {
+                view.encode(options.format, options, encodeCallback);
+            }
```

Comment:
Note to myself: the commit 7c947e5598f8e0a476230f5ec2ad92f0487b7bea is easier to understand. Actually on top of `v0.6.18`.